src/main/java/it/pagopa/ecommerce/commons/exceptions/NpgApiKeyMissingPspRequested.java

```diff
@@ -0,0 +1,26 @@
+package it.pagopa.ecommerce.commons.exceptions;
+
+import java.util.Set;
+
+/**
+ * Exception thrown when requesting an API key from NPG configuration for a
+ * nonexisting PSP
+ */
+public class NpgApiKeyMissingPspRequested extends Exception {
```

Comment:
```suggestion
public class NpgApiKeyMissingPspRequestedException extends Exception {
```
Why it is a checked exception? it can be a runtime exception and then make the NpgPspsApiKeysConfig.get method return an Either instead of explicitly throw exception. what do you think?

Reply:
Yeah, I was a little torn between making it checked or using `Either`

Reply:
Fixed in 3580a51